core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -850,6 +882,22 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
   protected def doKillExecutors(executorIds: Seq[String]): Future[Boolean] =
     Future.successful(false)
 
+  /**
+   * Request that the cluster manager decommission all executors on a given host.
+   * @return whether the decommission request is acknowledged.
+   */
+  final override def decommissionExecutorsOnHost(host: String): Boolean = {
+    logInfo(s"Requesting to kill any and all executors on host ${host}")
```

Comment:
nit. `${host}` -> `$host`?

Reply:
this matches the killExecsonHost code, so I'd either change both or not.

Reply:
+1 to change both

Reply:
also, "kill any executors" or "kill all executors" should be enough?

Reply:
this seems like bike shedding on a log message that I'd rather avoid.